src/main/java/com/st/lms/models/BookLoans.java

```diff
@@ -0,0 +1,93 @@
+package com.st.lms.models;
+
+import java.sql.Date;

```

Comment:
`java.sql.Date` is `java.util.Date` with added functionality; as Chris explained, `java.util.Date` is a broken API that should be used *only* for interacting with other APIs that require it (such as JDBC), *not* as data storage in your model classes.